ossapi/models.py

```diff
@@ -1073,6 +1073,19 @@ class ChatMessage(Model):
     # TODO enumify, example value: "plain"
     type: str
 
+class CountryStatistics(Model):
+    country: Country
+    country_code: str
+    created_at: Optional[Datetime]
+    display: int
+    id: int
+    mode: GameMode
+    performance: int
+    play_count: int
+    ranked_score: int
+    updated_at: Optional[Datetime]
+    user_count: int
+
```

Comment:
By my count, this is what `CountryStatistics` looks like:

```python
class CountryStatistics(Model):
    code: str
    active_users: int
    play_count: int
    ranked_score: int
    performance: int
    country: Country
```

Reply:
For having never looked at php before it was a good attempt. I trust you here way more than myself